### ISSUE TITLE
cssnano `calc` will break the Radio style

### ISSUE THREAD

**Issue by djyde** [adobe/react-spectrum#961]
It's not a bug. But I spend some time figure out how thing was happend. Hope this issue could save someone's time.

I found when I cssnano my css, the `Radio` will become:

![image](https://user-images.githubusercontent.com/914329/90231529-e7bd1c80-de4d-11ea-8cfa-8b5269f3339d.png)

And I tried to disable some cssnano optimization option and finally figure out is the [`calc`](https://cssnano.co/optimisations/calc) rule break this.

Different part:

![image](https://user-images.githubusercontent.com/914329/90231963-86e21400-de4e-11ea-80b0-7a62bd47feff.png)

**Comment by snowystinger** [comment]
We've had some issues with cssnano in the past, you might try a different version as well. Or the upstream postcss-calc.
I don't recognize this particular one, it might be helpful if you could log a bug with one of those projects

**Comment by devongovett** [comment]
😉 https://github.com/adobe/react-spectrum/blob/main/cssnano.config.js

**Comment by mischnic** [comment]
`postcss-calc` issue: https://github.com/postcss/postcss-calc/issues/107

**Comment by itaylor** [comment]
Might be worth noting that this happens to apps built using `create-react-app` out of the box, but only in `production` mode, not developer mode.  I tried, but couldn't find a way to change the config without ejecting the create-react-app config and modifying the webpack configuration.  

If you're using cra want to work the bug, around first `yarn eject` your create-react-app, then modify the `config/webpack.config.js` file. Change the settings for the default preset of `cssProcessorPluginOptions`  under the `OptimizeCSSAssetsPlugin` to include `calc: false,` 

Here's what my fixed version of that section looks like:
```js
cssProcessorPluginOptions: {
  preset: ['default', { calc: false, minifyFontValues: { removeQuotes: false } }],
},
```

**Comment by mischnic** [comment]
I've opened a PR for postcss-calc to fix this: https://github.com/postcss/postcss-calc/pull/114

**Comment by mischnic** [comment]
I've just verified that this is fixed with cssnano 7.0.4. Bump your lockfile
